### ISSUE TITLE
Expected emotions to detect

### ISSUE THREAD

**Issue by MotherJammer** [sajjjadayobi/FaceLib#29]
Hi! First I want to say thank you for your work in face detection and facial expression.
About the second topic, I have a question that I would gladly appreciate you answer:

What range of emotions is able to detect your implementation? The basic ones from Paul Ekman? (happy, sad, anger, disgust, surprise, fear and contempt) or another selection of emotions?

If you could explain as much as you can about the implementation for facial expression would be really helpful for me.

**Comment by ShannonZ** [comment]
https://github.com/sajjjadayobi/FaceLib/blob/master/facelib/FacialExpression/FaceExpression.py#L8

And the dataset infos:

> The task is to categorize each face based on the emotion shown in the facial expression into one of seven categories (0=Angry, 1=Disgust, 2=Fear, 3=Happy, 4=Sad, 5=Surprise, 6=Neutral). The training set consists of 28,709 examples and the public test set consists of 3,589 examples.
